src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/RecallsIntegrationTest.kt

```diff
@@ -132,7 +133,7 @@ class RecallsIntegrationTest : IntegrationTestBase() {
       .jsonPath("$.recallLength").isEqualTo("FOURTEEN_DAYS")
       .jsonPath("$.agreeWithRecallRecommendation").isEqualTo(agree)
       .jsonPath("$.lastReleasePrison").isEqualTo(prison)
-      .jsonPath("$.recallEmailReceivedDateTime").isNotEmpty
+      .jsonPath("$.recallEmailReceivedDateTime").value(endsWith("Z"))
```

Comment:
Do we have a test for this in this build for anything more specific than `endsWith("Z")`?
I'd like us to have one, though I know that can get a bit circular.

Reply:
AFAIAA, this is the only place in the code where we validate the json. This uses the random data generation. It is possible we should get rid of that, but I didn't want to do that as part of this ticket. I did read the PACT docs that also said you should avoid using random data, so it could be that we fix the test data and then we can assert on it. Happy to raise a ticket. @mikedunfordEE you raised the same thing